DESCRIPTION

```diff
@@ -1,6 +1,6 @@
 Package: corrr
 Type: Package
-Version: 0.2.1.9000
+Version: 0.3.0.9999

```

Comment:
Please change this back to `0.2.1.9000`. I tend to work as follows:

- Start with dev version (ending .9000) and remain on it until a CRAN release
- For CRAN release change to next version (e.g., `0.3.0`) 
- Once on CRAN, flag release on GitHub and change to dev version on next change (by appending `.9000`)

I tend to follow advice [here](http://r-pkgs.had.co.nz/release.html#release-version).

Which reminded me, the next version should actually be `1.0.0` (due to backward compatible issue). Regardless, we'll wait until ready for a CRAN release.